sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_sentiment.py

```diff
@@ -619,8 +619,8 @@ def test_opinion_mining_with_negated_opinion(self, client):
         document = client.analyze_sentiment(documents=documents, show_opinion_mining=True)[0]
 
         for sentence in document.sentences:
-            food_aspect = sentence.mined_opinions[0].aspect
-            service_aspect = sentence.mined_opinions[1].aspect
+            food_aspect = sentence.mined_opinions[0].target
```

Comment:
nit: food_target

Reply:
Updated here and also in test_analyze_sentiment_async.py.